min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
we used debian as reference - because it gives a good indication of support in other distros - particularly ubuntu.

If the deps have to be compiled to install the py libs, i would say that was a major blocker